bbb-common-web/src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+
+    <session-factory>
+
+        <!-- JDBC Database connection settings -->
+        <!--        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>-->
+        <!--        <property name="connection.url">jdbc:mysql://localhost:3306/bbb?useSSL=false&amp;serverTimezone=UTC</property>-->
+        <property name="connection.url">jdbc:postgresql://localhost:5432/bbb</property>
+        <property name="connection.username"></property>
+        <property name="connection.password"></property>
+
+
+        <!-- JDBC connection pool settings ... using built-in test pool -->
+        <property name="connection.pool_size">1</property>
+
+        <!-- Select our SQL dialect -->
+        <!--        <property name="dialect">org.hibernate.dialect.MySQLDialect</property>-->
+        <property name="dialect">org.hibernate.dialect.PostgreSQL92Dialect</property>

```

Comment:
Not sure why using 9.2 Dialect. The newer version of PostgreSQL brings some useful SQL standards integration that we might need.

Reply:
Even if we want to support running this directly on a BBB server rather than in docker, Ubuntu 18.04 ships with PostgreSQL 10, so that would be a safe minimum version to use.